lib/Doctrine/DBAL/Schema/Table.php

```diff
@@ -852,12 +852,8 @@ private function _createIndex(array $columns, string $indexName, bool $isUnique,
             throw IndexNameInvalid::new($indexName);
         }
 
-        foreach ($columns as $index => $value) {
-            if (is_string($index)) {
-                $columnName = $index;
-            } else {
-                $columnName = $value;
-            }
+        foreach ($columns as $value) {
+            $columnName = $value;
```

Comment:
Can we get rid of this intermediary variable? `foreach ($columns as $columnName)`

Reply:
You're right, totally missed it! Fixed.